app/objects/beatmap.py

```diff
@@ -47,8 +47,8 @@ async def api_get_beatmaps(**params: Any) -> Optional[list[dict[str, Any]]]:
         url = "https://old.ppy.sh/api/get_beatmaps"
         params["k"] = str(app.settings.OSU_API_KEY)
     else:
-        # https://doc.kitsu.moe/
-        url = "https://kitsu.moe/api/get_beatmaps"
+        # https://osu.direct/doc
+        url = "https://osu.direct/api/get_beatmaps"
```

Comment:
is it backwards compatible? seems like https://kitsu.moe/api/get_beatmaps goes to https://osu.direct/home

Reply:
this should be pretty easy to make 100% backwards compatible with some rerouting rules,
`https://kitsu.moe/$1` -> `https://osu.direct/$1`

Reply:
they seemed to have hard redirected to /home always, it broke akatsuki. this sucks :(

Reply:
@HorizonCode we recently had this problem when moving from akatsuki.pw to akatsuki.gg

if you're using cloudflare, consider setting up page rules like this for this (and future domain switches)
![image](https://i.cmyui.xyz/IjZnuySmjYMK.png)

Reply:
i will add that 👍